taskcluster/ci/ui-test/kind.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         run-on-git-branches: ["^((?!releases[_/]).+)$"]
         run:
             commands:
-                - [automation/taskcluster/androidTest/ui-test.sh, x86, app.apk, android-test.apk, '50']
+                - [automation/taskcluster/androidTest/ui-test.sh, x86, app.apk, android-test.apk, '-1']
```

Comment:
I think this needs a slight bit of housekeeping.  If we're using a config file (the flank file) to contain our variables, why should this one stand out as the single variable we need to pass down?  I can't remember what the logic was for leaving it this way, but we should be invoking this variable from the flank file (and not in both places).  I think you could simply remove it from here and from line #137 and then I believe it would just be picked up from the flank file automagically.

It would probably also be a good idea to make sure the help is up-to-date: https://github.com/mozilla-mobile/fenix/blob/master/automation/taskcluster/androidTest/ui-test.sh#L24-L33

Aside from that nit, LGTM

Reply:
Once that change is in place, pls land this.  thanks!